ci/test.sh

```diff
@@ -61,11 +61,16 @@ export CODACY_PROJECT_TOKEN="$(cat /workspace/ci/codacy-project-token | sed -e '
 curl -Ls https://coverage.codacy.com/get.sh > get.sh
 chmod +x get.sh
 for f in \$(find . -type f -name 'jacoco.xml'); do
+    echo Processing \$f
```

Comment:
How does it help? Can you elaborate?

Reply:
My theory is to add delays between individual upload calls to reduce overall usage rate. 

This particular echo message has nothing to do with the base problem. I added it only for more observability since it's hard to know which 3rd-party message belongs to which tool invocation.